docs/src/pages/components/selects/selects.md

```diff
@@ -25,6 +25,9 @@ It's meant to be an improved version of the "react-select" and "downshift" packa
 
 ## Props
```

Comment:
We don't use this heading in other docs pages. Should we?

Reply:
Should we start using a section with `## Props`? Not sure, this doesn't feel very problem oriented.

Reply:
Then we should remove it here?

Reply:
No objections

Reply:
That's probably a topic for another effort. Note that this structure is specific to v5, so we have recently improved it.